lib/FrontendTool/FrontendTool.cpp

```diff
@@ -929,8 +939,6 @@ static bool performCompile(CompilerInstance &Instance,
       debugFailWithCrash();
   }
```

Comment:
Arguably these crashes should happen even in -parse or -resolve-imports.

Reply:
For future reference: Had an offline chat about this.  [Filed SR-8252](https://bugs.swift.org/browse/SR-8252) about it.